src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('tbody');
+
+const listHeader = [ ...document.querySelectorAll('th') ];
+
+listHeader.map((item, index) => {
+  item.addEventListener('click', () => {
+    const rows = [ ...list.querySelectorAll('tr') ];
+
+    rows.sort((a, b) => {
+      const itemA = a.children[index].innerHTML.replace(/[^A-Z0-9]/gi, '');
```

Comment:
if you're looking for the text content of the node it's more appropriate to use textContent or innerText props